src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.io.*;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        StringBuilder sb = new StringBuilder();
+        int buy = 0;
+        int supply = 0;
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            String line = br.readLine();
+            while (line != null) {
+                String operation = line.split(",")[0];
+                int quantity = Integer.parseInt(line.split(",")[1]);
+                line = br.readLine();
+                switch (operation) {
+                    case "buy":
+                        buy += quantity;
+                        break;
+                    case "supply":
+                        supply += quantity;
+                        break;
+                    default:
+                        System.out.println("Invalid operation");
```

The `default` branch prints "Invalid operation" and continues. This silently ignores unrecognized operations. The checklist's guidance "Don't ignore exceptions" requires controlled failures for unexpected input. Replace the println with a RuntimeException (including the file name and offending line/operation) so tests can detect the invalid input.